src/ints/int10_vesa.cpp

```diff
@@ -576,10 +576,6 @@ void INT10_SetupVESA(void) {
 	for (i=0;i<len;i++) {
 		phys_writeb(0xc0000+int10.rom.used++,string_oem[i]);
 	}
-	switch (svgaCard) {
-	case SVGA_S3Trio:
-		break;
-	}
```

Comment:
Very odd; I suppose the compile has been eliding this away for years!

Reply:
Yeah, this is weird - as if someone planned to implement something, or left refactoring halfway done… (this generated 4 warnings IIRC).

Reply:
That's like breaking the sound barrier with code.. achieved more warnings than lines of code.